Simplified/Reader2/Bookmarks/NYPLAnnotations.swift

```diff
@@ -69,16 +76,16 @@ final class NYPLAnnotations: NSObject, NYPLAnnotationSyncing {
   ///   process, unless sync is not supported by the current library.
   @objc
   class func requestServerSyncStatus(forAccount userAccount: NYPLUserAccount,
```

Comment:
good point. This work was part of IOS-382 but I decided not to do it because the way we are headed is to have hybrid to the work that NYPLAnnotations is doing (i.e. network calls for bookmarks etc). I did file PJR-90 though for removing explicit dependencies on NYPLUserAccount.